appcontainer/ViewportSizeModel.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {HoistModel} from '@xh/hoist/core';
+import {action, computed, observable, makeObservable} from '@xh/hoist/mobx';
+
+/**
+ * Track observable width / height of the browser viewport, and provide observable
+ * access to device orientation
+ *
+ * @private
+ */
+export class ViewportSizeModel extends HoistModel {
+
+    /** @member {Number} */
+    @observable width;
```

Comment:
since these change atomically and we are only actually revealing "size" let's just simplify things by having a single @observable.ref property that is the object.  Probably doesn't matter, but its going to be less code/ observables and complexity, and this thing may actually change very rapidly when window is being resized.